### ISSUE TITLE
Error page doesn't look good

### ISSUE THREAD

**Issue by drmenzelit** [joomla/cassiopeia#100]
Menu is hamburger only
Logo doesn't appear

![image](https://user-images.githubusercontent.com/9153168/93871164-800cb380-fcce-11ea-9bb5-8a1d6ace3752.png)

**Comment by chmst** [comment]
https://issues.joomla.org/tracker/joomla-cms/30729

**Comment by ghost** [comment]
similaer https://github.com/joomla/joomla-cms/issues/30023

**Comment by richard67** [comment]
After #101 has been merged, is there still something to be done for this issue?

**Comment by drmenzelit** [comment]
The problem was solved with PR #101 
We can open a new issue for the structure and content of the error page.
